internal/core/analysis_api/handlers/list.go

```diff
@@ -78,7 +78,9 @@ func pythonListFilters(
 	enabled *bool,
 	nameContains string,
 	severity []compliancemodels.Severity,
-	types, tags []string,
+	types,
+	tags []string,
```

Comment:
Nit: if these are on separate lines, but the type twice so it's easier to read:

```suggestion
	types []string,
	tags []string,
```